### PR TITLE
more stuff

### PR DESCRIPTION
**Pull request by shihyuntang**
testing...

### CODE REVIEW

File: sytang/HW1.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import matplotlib.pyplot as plt
+
+
+# Fixing random state for reproducibility
+np.random.seed(19680801)
+
+# Compute areas and colors
+N = 150
+r = 2 * np.random.rand(N)
+theta = 2 * np.pi * np.random.rand(N)
```
Comment (cjtu) [inline sytang/HW1.py:14]: Comment here

File: sytang/HW1.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+# delta1 = np.diff(price_data.adj_close) / price_data.adj_close[:-1]
+
+# # Marker size in units of points^2
+# volume = (15 * price_data.volume[:-2] / price_data.volume[0])**2
+# close = 0.003 * price_data.close[:-2] / 0.003 * price_data.open[:-2]
+
+# fig, ax = plt.subplots(figsize=(5,3), dpi=200)
+# ax.scatter(delta1[:-1], delta1[1:], c=close, s=volume, alpha=0.5)
+
+# ax.set_xlabel(r'$\Delta_i$', fontsize=15)
```
Comment (cjtu) [inline sytang/HW1.py:47]: Why this

**Review by cjtu — COMMENTED** [review]
Nice work!

File: sytang/HW1.py (hunk 1 above)
Comment (shihyuntang) [reply sytang/HW1.py:14]: ok...... roger that

**Pull request closed without merging**